rust/storage/src/lib.rs

```diff
@@ -316,6 +316,12 @@ impl DefaultDispatcher {
         }
     }
 
+    /// Stores an already existing Vts structure.
+    pub fn set_vts(&self, vts: Vts) -> Result<(), StorageError> {
```

Comment:
wouldn't it bet better to transform a given reader into Vts instead of letting the caller do that?

Reply:
Can be done within another PR